src/App.js

```diff
@@ -0,0 +1,152 @@
+import { useState } from "react";
```

Comment:
One comment for the whole file - it's usually better to separate components into different files. For example, `Square`, `Board` etc. would have their own file for this case.

After creating the file, you need to export those like you for `Game` component.